cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -13,9 +13,11 @@
     v1_to_v2_name_map,
     unused_attributes_list,
     convert_to_partners_map,
+    attributes_move_to_composition,
 )
 
 encoder.FLOAT_REPR = lambda o: format(o, ".8g")
+CURRENT_VERSION = "2.0.0"
```

Comment:
Let's change this to "2.0", and keep all the format version just with 1 decimal place.